src/components/OcAppSideBar.vue

```diff
@@ -64,6 +58,13 @@ export default {
 .oc-app-side-bar {
   height: 100vh;
 }
+
+.oc-app-side-bar-header {
+  background-color: var(--oc-color-background-default);
+  position: sticky;
+  top: 0px;
```

Comment:
Sticky position needs the actual top position i.e. in web under the top bar...

Reply:
With the current state this can be ignored as we do not do "real" scrolling of the window but scroll containers instead